go.mod

```diff
@@ -116,6 +118,8 @@ require (
 	github.com/prometheus/procfs v0.0.0-20190519111021-9935e8e0588d // indirect
 	github.com/shirou/gopsutil v2.18.12+incompatible
 	github.com/syndtr/goleveldb v1.0.0
+	github.com/tyler-smith/go-bip32 v0.0.0-20170922074101-2c9cfd177564
+	github.com/tyler-smith/go-bip39 v1.0.0
```

Comment:
Where do you use these modules?

Reply:
Actually these come from another ticket for seed phrase, might missing from there. @elluck91